docs/guides/core_client.md

```diff
@@ -425,6 +425,22 @@ $ cargo run -- -f <path-to-toml-config-file> crs-gen-result --request-id <REQUES
 
 Upon success, both the command to request to generate a CRS _and_ the command to fetch the result, will save the CRS produced by the core in the `object_folder` given in the configuration file.
 
+#### Backup restoring
+
+If a backup vault is specified in the in the server configuration toml file, then all non-volatile private key material (i.e. what is stored in the private vault) is backed up to this location. This also means that it is possible to restore this content in case access to the private vault is lost, or that the private vault needs to be moved.
+This is done through the backup recovery command:
+
+```{bash}
+$ cargo run -- -f <path-to-toml-config-file> custodian-backup-restore
+```
+
+Note that this operation will copy the content from the backup vault to the private vault. In case any of the backed up content already exists in the private vault, then the request will fail.
```

Comment:
I don't believe this is correct. I believe that existing files will simply be ignored, i.e. nothing gets overwritten, but what is not there, gets written.